content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -87,7 +87,7 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
   zero. Its semantics differs according to the value of `whenUnsatisfiable`:
   - when `whenUnsatisfiable` equals to "DoNotSchedule", `maxSkew` is the maximum
     permitted difference between the number of matching pods in the target
-    topology and the global minimum.
+    topology and the global minimum (global minimum is the minimum number of pods in all topology domains).
```

Comment:
> global minimum is the minimum number of pods in all topology domains

Is this global minimum a singleton value at the cluster level, or is it “global” because it's across multiple topology domains but also specific to the label selector being used?

(I assume the latter, but let's make it clear)

Reply:
You are right that it's the latter. Updated the doc.